### DISCUSSION TITLE
Add a way to sign up for discussions instead of us spamming you

### DISCUSSION THREAD

**Discussion by ghost** [community/community#2830] [Discussions] [answered]
Right now, a lot of people are spamming the nice people that work on GitHub to get into discussions. While I could be wrong, I feel like there could be a better way to do this. Maybe in the settings tab somewhere that says sign up for beta (not necessarily turn it on, more like sign up for an opportunity to get it)? I’m sure there’s a reason why you haven’t done this, but it feels like it’s been a while since this launched into beta and maybe some more feedback might be appreciated. 

Who knows, I could be just talking about some idea that’s been considered and decided to hold off for now, but maybe even showing it only to the bigger repos is a good idea. Once again, who knows.

**Comment by becca** [comment, marked as answer]
It is now possible for you to enable yourself on public repositories!
https://github.blog/changelog/2020-12-08-github-discussions-now-available-as-a-public-beta/

**Reply by ghost** [reply]
This along with dark mode is amazing! Great work to you and all the people that make GitHub great!
